3_navigation.ipynb

```diff
@@ -399,9 +399,9 @@
    },
    "source": [
     "## Send a goal\n",
-    "To send a goal to the robot, execute the following command and observe the movement of the robot on the simulator.\n",
+    "To send a goal to the robot, execute the following command and observe the movement of the robot in the simulator.\n",
     "\n",
-    "**For advanced user** To move the move base autonomously, use the function `move_base_goal`. If you are interested, please take a look:\n",
+    "**For advanced user**: To move the move base autonomously, use the function `move_base_goal`. If you are interested, please take a look:\n",
```

Comment:
"For advanced user" should really be "For advanced users".
I must not have specified it before.  Sorry.